### ISSUE TITLE
Aseprite hot reloading crash?

### ISSUE THREAD

**Issue by deepnight** [deepnight/ldtk#485]

**Comment by chamalowmoelleux** [comment]
I had the crash, just saved changes in my Aseprite file (no resizing or anything, I just changed pixels) and LDtk crashed trying to reload my file.

Here is my crash report, and a screenshot of the crash screen:
![crash report](https://user-images.githubusercontent.com/49991411/127636001-db9c9adc-5fef-4e7d-a333-c2218a9898bb.png)

Stack:
```
LDtk version: 0.9.3-64bits
check
Error

```
Processes:
```
#0 App
 |--- #1 page.Editor
       |--- #2 misc.FileWatcher
       |--- #3 display.WorldRender
       |--- #4 display.LevelRender
       |--- #5 display.Camera
       |--- #6 display.Rulers[--]
       |--- #7 tool.SelectionTool[--]
       |--- #8 tool.lt.DoNothing[--]
       |--- #9 WorldTool
       |--- #10 tool.PanView[--]
       |--- #53 tool.lt.IntGridTool[--]
       |--- #180 ui.modal.panel.EditAllAutoLayerRules
       |--- #240 ui.Notification
```

Log:
```
[EVENT]        LayerRuleAdded 44
[FILE]         Reading HTML template ruleEditor from cache
[EVENT]        ToolOptionChanged
[EVENT]        LayerRuleChanged 44
[EVENT]        LayerRuleChanged 44
[EVENT]        LayerRuleChanged 44
[EVENT]        LayerRuleChanged 44
[GENERAL]      Added rule
[EVENT]        LayerRuleAdded 45
[FILE]         Reading HTML template ruleEditor from cache
[EVENT]        ToolOptionChanged
[EVENT]        ToolOptionChanged
[EVENT]        ToolOptionChanged
[EVENT]        LayerRuleChanged 45
[EVENT]        LayerRuleChanged 45
[EVENT]        LayerRuleChanged 45
[EVENT]        LayerRuleChanged 45
[EVENT]        LayerRuleChanged 45
[GENERAL]      Added rule
[EVENT]        LayerRuleAdded 46
[FILE]         Reading HTML template ruleEditor from cache
[EVENT]        ToolOptionChanged
[EVENT]        ToolOptionChanged
[EVENT]        LayerRuleChanged 46
[EVENT]        LayerRuleChanged 46
[EVENT]        LayerRuleChanged 46
[EVENT]        LayerRuleChanged 46
[EVENT]        LayerRuleChanged 46
[FILE]         Changed on disk: C:\Users\UserName\Documents\trucs à moi\Pixels Arts\Tilesets\gne.aseprite
[CACHE]        Reloading cached image gne.aseprite...
[CACHE]        Disposing cached image gne.aseprite...
[CACHE]        Caching image gne.aseprite...
[CACHE]         -> identified as Aseprite
[ERROR]        Failed to decode pixels: gne.aseprite (identified as Aseprite, err=TypeError: Cannot read property 'dispose' of null)
[FILE]         Reloading tileset: Gne path=gne.aseprite
[FILE]         Loading atlas image: gne.aseprite...
[CACHE]        Caching image gne.aseprite...
[CACHE]         -> identified as Aseprite
[ERROR]        Failed to decode pixels: gne.aseprite (identified as Aseprite, err=TypeError: Cannot read property 'dispose' of null)
[ERROR]        Image loading failed
[FILE]          -> Reload result: LoadingFailed(Unknown error)
[FILE]          -> pixelData: need rebuild
[GENERAL]      Page started: page.CrashReport()
[FILE]         Loading page template: crashReport from C:/Users/UserName/AppData/Local/Programs/ldtk/resources/app.asar/assets/tpl/pages/crashReport.html
[ERROR]        check (Error)
[ERROR]        undefined

[GENERAL]      
#0 App
 |--- #1 page.Editor
       |--- #2 misc.FileWatcher
       |--- #3 display.WorldRender
       |--- #4 display.LevelRender
       |--- #5 display.Camera
       |--- #6 display.Rulers[--]
       |--- #7 tool.SelectionTool[--]
       |--- #8 tool.lt.DoNothing[--]
       |--- #9 WorldTool
       |--- #10 tool.PanView[--]
       |--- #53 tool.lt.IntGridTool[--]
       |--- #180 ui.modal.panel.EditAllAutoLayerRules
       |--- #240 ui.Notification
 |--- #241 page.CrashReport

[FILE]         Cleared all file watches
```

**Comment by chamalowmoelleux** [comment]
Oh, I think I found the issue, LDtk can't load an Aseprite File who contains a empty layer, and the changes I made before the crash was deleting all the pixels on a layer. After this, it didn't wanted to load my Aseprite file until I placed some pixels on the empty layer.

**Comment by deepnight** [comment]
Fixed in 0.9.4
